src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md

```diff
@@ -5,46 +5,76 @@ head_title: 'xk6-disruptor Documentation'
+Currently, the disruptor is intended to test applications running in Kubernetes. Other platforms are not supported at this time.
 
-The main use case for xk6-disruptor is to test how resilient an application is to diverse types of disruptions. xk6-disruptor can reproduce the effects of these disruptions, without having to reproduce their root causes.
-For example, you can inject delays in the HTTP requests that an application makes to a service  without having to stress or interfere with the infrastructure (network, nodes) on which the service runs.
+It provides a Javascript API to inject different [faults](/javascript-api/xk6-disruptor/api/faults/) in HTTP and gRPC requests, such as errors and delays, into the selected Kubernetes [Pods](javascript-api/xk6-disruptor/api/poddisruptor) or [Services](/javascript-api/xk6-disruptor/api/servicedisruptor).
 
-In this way, xk6-disruptor makes reliability tests repeatable, predictable, and with a limited blast radius.
-These characteristics make it easier to incorporate such tests into the test suites of applications that deploy on shared infrastructures (such as staging environments).
+Other types of faults and disruptors will be introduced in the future. The [Roadmap](https://github.com/grafana/xk6-disruptor/blob/main/ROADMAP.md) presents the project's goals for the coming months regarding new functionalities and enhancements.
 
+```javascript
+export default function () {
+    // Create a new pod disruptor with a selector
+    // that matches pods from the "default" namespace with the label "app=my-app"
+    const disruptor = new PodDisruptor({
+        namespace: "default",
+        select: { labels: { app: "my-app" } },
+    });
 
-<Blockquote mod="attention">
+    // Check that there is at least one target
+    const targets = disruptor.targets();
+    if (targets.length === 0) {
+        throw new Error("expected list to have one target");
+    }
 
-xk6-disruptor is in the alpha phase, undergoing active development. k6 doesn't guarantee API compatibility between releases.
-Until this extension reaches v1.0 release, your k6 scripts may need to be updated on each release .
+    // Disrupt the targets by injecting HTTP faults into them for 30 seconds
+    const fault = {
+        averageDelay: 500,
+        errorRate: 0.1,
+        errorCode: 500
+    }
+    disruptor.injectHTTPFaults(fault, "30s")
+}
+```
 
-</Blockquote>
+## Use cases
 
-## Try out xk6-disruptor on our demo environment
+The disruptor lets you test the resiliency of distributed applications by introducing errors in the requests served by your services.
 
-We provide an [interactive demo environment in Killercoda](https://killercoda.com/grafana-xk6-disruptor/scenario/killercoda) which you can use to try xk6-disruptor right away without having to install Kubernetes on your local machine.
+The disruptor does not try to reproduce root causes, such as failed application instances or degraded computing or network resources. 
```

Comment:
I would perhaps join these two paragraphs into one, to avoid repeating "The disruptor".

Reply:
Need to merge this PR. @heitortsergent, please, consider it for later changes.